docs/modules/eventbus.md

```diff
@@ -0,0 +1,26 @@
+# eventbus
+## **Overview** 
+Eventbus acts as an interface for sending/receiving messages on mqtt topics. It supports 3 kinds of mode: internalMqttMode,externalMqttMode and BothMqttMode.
+
+## eventbus suscribes to the following topics
```

Comment:
The heading could be "Topics"

and then below that , we can have this line . 
also the topics can be put in a  list format.

Reply:
fixed